api/src/API/Controllers/RaceResultsController.cs

```diff
@@ -38,12 +39,28 @@ public async Task<IActionResult> GetAllRaceResults(string orgId, string memberId
         [HttpGet("/organizations/{orgId}/raceresults")]
         public async Task<IActionResult> GetAllRaceResults(string orgId)
         {
+            RaceContainerClient raceContainer = containerProvider.RaceContainer;
             MemberContainerClient memberContainer = containerProvider.MemberContainer;
+
             IEnumerable<Member> members = await memberContainer.GetAllMembersAsync(orgId);
             IEnumerable<string> memberIds = members.Select(member => member.Id.ToString());
 
             RaceResultContainerClient container = containerProvider.RaceResultContainer;
-            IEnumerable<RaceResult> result = await container.GetRaceResultsForMembersAsync(memberIds);
+            IEnumerable<RaceResult> raceResults = await container.GetRaceResultsForMembersAsync(memberIds);
+
+            var racesNeeded = raceResults.Select(result => result.RaceId).ToHashSet();
+            var membersNeeded = raceResults.Select(result => result.MemberId);
```

Comment:
Isn't this the same as the line above? We're getting all of the race results for a set of members and then getting the members for those race results?

Reply:
Yes, but aren't we already fetching their record? We're getting all members for an org already.

Reply:
good catch, didn't realize we were doing that